test/Microsoft.DotNet.ProjectModel.Tests/LockFilePatchingTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Microsoft.DotNet.Tools.Test.Utilities;
+
+namespace Microsoft.DotNet.ProjectModel.Tests
+{
+    public class LockFilePatchingTests : TestBase
+    {
+
+        private string ExportFilesRoot=> Path.Combine(RepoRoot, "TestAssets", "LockFileSamples", "ExportFileSamples");
+
+        [Fact]
+        public void TestValidPatching()
```

Comment:
The relationship between this test and Patching is very unclear. This test seems to be `WhenLockFileIsValidThenExportFileRepresentsItsContents`. Is patching just an implementation detail that is not being directly evaluated?